libs/core/src/lib/time-picker/time-picker.component.html

```diff
@@ -29,12 +28,14 @@
     </fd-popover-control>
     <fd-popover-body *ngIf="displaySeconds || displayMinutes || displayHours">
         <fd-time
+            [tablet]="tablet"
+            [compact]="compact"
             [disabled]="disabled"
             [meridian]="meridian"
             [(ngModel)]="time"
             [keepTwoDigits]="keepTwoDigitsTime"
+            (focusFirstElement)="inputElement.focus()"
```

Comment:
Maybe a non-issue but should shift+tab when the input is focused, then focus the button in the time component?

Reply:
I will plug focus-trap here.